sharing/resource_share.go

```diff
@@ -58,6 +58,12 @@ type ShareUpdates struct {
 	Updates []ShareDataChange `json:"updates"`
 }
 
+func (su *ShareUpdates) sortSharesByName() {
```

Comment:
does the API expect the list of updates to be sorted?
```suggestion
func (su *ShareUpdates) sortUpdatesByName() {
```

Reply:
This is for consistency, we sort them in get. Also, this is required for our unit tests (expected request is a list and we use set now).